<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTriangleDialog</class>
 <widget class="QDialog" name="CreateTriangleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelB">
     <property name="text">
      <string>Сторона В</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelC">
     <property name="text">
      <string>Сторона С</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelA">
     <property name="text">
      <string>Сторона А</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleTriangleBoxA"/>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleTriangleBoxB"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleTriangleBoxC"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateTriangleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
